libs/designsystem/package.json

```diff
@@ -6,14 +6,13 @@
     "@fontsource/roboto": "4.2.1",
     "@ionic/angular": "5.5.2",
     "highcharts": "8.0.4",
-    "moment": "2.24.0",
-    "tslib": "^2.0.0"
+    "moment": "2.24.0"
   },
   "peerDependencies": {
-    "@angular/common": ">=8.2.7",
-    "@angular/core": ">=8.2.7",
-    "@angular/forms": ">=8.2.7",
-    "@angular/router": ">=8.2.7",
+    "@angular/common": ">=11.2.5",
+    "@angular/core": ">=11.2.5",
+    "@angular/forms": ">=11.2.5",
+    "@angular/router": ">=11.2.5",
```

Comment:
On second thought, I'm not sure this is actually required?
I see[ Ionic is only requiring >=8.2.7](https://github.com/ionic-team/ionic-framework/blob/master/angular/package.json#L48-L54)

Reply:
Ok. Should we revert it then?